src/controllers/update/update.interrupt.card.controller.ts

```diff
@@ -3,11 +3,20 @@ import { NextFunction, Request, Response } from "express";
 import { logger } from "../../utils/logger";
 import * as config from "../../config";
 import { isActiveFeature } from "../../utils/feature.flag";
+import { isRemoveJourney } from "../../utils/url";
 
 export const get = (req: Request, res: Response, next: NextFunction) => {
   try {
     logger.debugRequest(req, `${req.method} ${req.route.path}`);
 
+    if (isRemoveJourney(req)){
+      return res.render(config.UPDATE_INTERRUPT_CARD_PAGE, {
+        journey: config.JourneyType.remove,
+        backLinkUrl: (isActiveFeature(config.FEATURE_FLAG_ENABLE_UPDATE_TRUSTS) ? config.SECURE_UPDATE_FILTER_URL : config.UPDATE_ANY_TRUSTS_INVOLVED_URL) + config.JOURNEY_REMOVE_QUERY_PARAM,
```

Comment:
Should the backlink have the journey query param?

Reply:
Yes, I think it's needed, so that when we navigate back in the journey the receiving page knows that it's a 'remove' we're doing.

Reply:
I agree - is needed. Looks like it's missing on the SECURE_UPDATE_FILTER_URL which is the previous page when trusts are enabled.

Reply:
ok - ignore that - I've spotted the brackets

Reply:
Yes. Another feature flag 'getting in the way' and making the code a bit confusing...